File: Selenium/Activities/Activity2.py
from selenium import webdriver
from webdriver_manager.firefox import GeckoDriverManager
from selenium.webdriver.firefox.service import Service as FirefoxService
from selenium.webdriver.remote.webdriver import By
from selenium.webdriver.support.select import Select

service = FirefoxService(GeckoDriverManager().install())
with webdriver.Firefox(service = service) as driver:
    driver.get("https://www.training-support.net/selenium/login-form")
    print("The title of the page is: ", driver.title)
    driver.find_element(By.ID, "username").send_keys("admin")
    driver.find_element(By.ID, "password").send_keys("password")
    driver.find_element(By.XPATH, "//*[@id='dynamic-attributes-form']/div/button").click()
    message = driver.find_element(By.ID, "action-confirmation")
    print("confirmation message is: ", message.text)
    driver.quit()

File: Selenium/Activities/activity2.java
import io.github.bonigarcia.wdm.WebDriverManager;
import org.openqa.selenium.WebDriver;
import org.openqa.selenium.By;
import org.openqa.selenium.firefox.FirefoxDriver;

public class activity2 {
    public static void main(String[] args) {
        System.setProperty(FirefoxDriver.SystemProperty.BROWSER_LOGFILE, "/dev/null");
        WebDriverManager.firefoxdriver().setup();
        WebDriver driver = new FirefoxDriver();
        driver.get("https://www.training-support.net/selenium/login-form");
        System.out.println("The title of the page is: " + driver.getTitle());
        driver.findElement(By.id("username")).sendKeys("admin");
        driver.findElement(By.id("password")).sendKeys("password");
        driver.findElement(By.xpath("//*[@id=\"dynamic-attributes-form\"]/div/button")).click();
        System.out.println("Login is successful");
        String message = driver.findElement(By.id("action-confirmation")).getText();
        System.out.println("Login message is: " + message);
        driver.quit();
    }
}
